src/lib.rs

```diff
@@ -265,7 +265,29 @@ impl Build {
 
         log::info!("Successfully built in {}", build_dir_path.display());
 
-        Ok(build_dir_path)
+        Ok(
+            BuildResult::new(
+                build_dir_path,
+                wasm_js_path,
+                wasm_bin_path,
+            )
+        )
+    }
+}
+
+pub struct BuildResult {
+    pub build_dir_path: PathBuf,
```

Comment:
The suffix path is redundant with the suffix dir
```suggestion
    pub build_dir: PathBuf,
```